bin/container.sh

```diff
@@ -21,7 +21,6 @@ mysql=mysql:5.6
 docker_network=${project}_nw
```

Comment:
Are the changes here related to the feature or do they belong to the feature/docker branch? I have tested the feature itself with scripts from the other branch since they are working very nicely, so I am not sure about the changes here.